app/controllers/profiles_controller.rb

```diff
@@ -45,7 +45,7 @@ def dashboard
     @passengers = Passenger.where(journey_id: Journey.where(user: @user))
 
 
-    else
+    elsif @user.cars[0]
```

Comment:
That's OK but I wouldn't use it in this context.

```
car = user.cars.first
...
elsif car
...
```

If you call `.first` twice, you're hitting the DB twice. Therefore using `teacher?` and then `.first` wouldn't be optimal.